server/src/completion/YmlCompletionItemsProvider.ts

```diff
@@ -37,4 +37,18 @@ export class YmlCompletionItemsProvider {
             })
             .filter((elem) => !!elem);
     }
+
+    /**
+     * Find the *first* completion item that has `entityName` as name.
+     *
+     * @param entityName The name of the entity searched for.
+     * @returns The first completion item that has the same label as `entityName`, `null` otherwise.
+     */
+    public getItemByLabel(entityName: string): AbstractYmlObject {
+        const sameNameElements = this.completions.filter((elem) => elem.label === entityName);
+        if (!sameNameElements || sameNameElements.length === 0) {
```

Comment:
? I'd find it surprising if `filter(…)` could return `null` in some cases and an empty array / collection / whatever in other cases. :thinking:
:hourglass_flowing_sand: 

> A new array with the elements that pass the test. If no elements pass the test, an empty array will be returned.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter